docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,137 @@
+# Kubernetes Deployment
+
+## AnyCable-Go
+
+anycable-go can be easily deployed to your Kubernetes cluster using Helm and [our official Helm chart][anycable-helm].
+
+ 1. Add it as a dependency to your main application:
+
+    ```yaml
+    # Chart.yaml
```

Comment:
It's important to mention that it makes sense for Helm 3 only.

Reply:
Added note